### PR TITLE
chore: update baseline guide

### PR DESCRIPTION
**Pull request by ruheni**
## Describe this PR

TODO:
- [x] Merge "Add Prisma Migrate to a project" and "Baselining" guides
- [x] Update "Add Prisma Migrate to a project" baselining instructions to match instructions in [Getting started with an existing  tutorial](https://www.prisma.io/docs/getting-started/setup-prisma/add-to-existing-project/relational-databases/baseline-your-database-typescript-postgres)
- [ ] ~Point to new baselining information from the getting started guide~ Already points to guide
- [x] Create a redirect for "Baselining" guide to point to "Add Prisma Migrate to a Project" (header)
- [x] Update `--create-only` docs to mention reset prompt/ drift
- [x] Update Migration guides from other ORMs to baseline guide (handled in #4603)

### CODE REVIEW

File: content/400-reference/200-api-reference/200-command-reference.mdx [hunk 1]
```diff
@@ -936,6 +936,12 @@ See also:
 | `--skip-generate` | No       | Skip triggering generators (for example, Prisma Client)                                                                             |         |
 | `--help` / `--h`  | No       | Displays the help message                                                                                                           |
 
+<Admonition type="info">
+
+If a [schema drift](/concepts/components/prisma-migrate/shadow-database#detecting-schema-drift) is `prisma migrate dev` detected while using `--create-only`, you will be prompted to reset your database.
```
Comment (janpio) [inline content/400-reference/200-api-reference/200-command-reference.mdx]: Sentence is garbled.

File: content/300-guides/050-database/100-developing-with-prisma-migrate/050-add-prisma-migrate-to-a-project.mdx [hunk 2]
```diff
@@ -33,23 +37,47 @@ Make sure your Prisma schema is in sync with your database schema. This should a
...
+   --from-empty \
+   --to-schema-datamodel prisma/schema.prisma \
+   --script > prisma/migrations/0_init/migration.sql
    ```
-   ./prisma/migrations/20210426141759_initial-migration
-   ```
+
+1. Review the generated migration
 
 ### Work around features not supported by Prisma Schema Language
```
Comment (janpio) [inline content/300-guides/050-database/100-developing-with-prisma-migrate/050-add-prisma-migrate-to-a-project.mdx]: Does this section still make sense in this context?
Do we have this in all the other places where we explain to use baselining?

File: content/400-reference/200-api-reference/200-command-reference.mdx (hunk 1 above)
Comment (ruheni) [inline content/400-reference/200-api-reference/200-command-reference.mdx]:
```suggestion
If a [schema drift](/concepts/components/prisma-migrate/shadow-database#detecting-schema-drift) is detected while running `prisma migrate dev` using `--create-only`, you will be prompted to reset your database.
```

File: content/300-guides/050-database/100-developing-with-prisma-migrate/050-add-prisma-migrate-to-a-project.mdx (hunk 2 above)
Comment (ruheni) [reply content/300-guides/050-database/100-developing-with-prisma-migrate/050-add-prisma-migrate-to-a-project.mdx]: We don't. I'll take another look at it and make any necessary updates.

Comment (janpio) [reply content/300-guides/050-database/100-developing-with-prisma-migrate/050-add-prisma-migrate-to-a-project.mdx]: It's fine if we have 1 location that shows the full complexity, and the other places that use the same process skip over some of the details.

**Review by sabinadams — APPROVED** [review]

File: content/800-data-platform/400-members/100-add.mdx [hunk 3]
```diff
@@ -27,18 +27,19 @@ In Prisma Data Platform, you invite team members to a project. To invite a team
...
+On the [Scale plan](/data-platform/billing/plans-and-quotas#strongscalestrong-plan), you can assign any of the [available roles](/data-platform/members/roles-permissions).
 
   </Admonition>
 
-  <img
-    src="images/team-members-invite-and-select-role.png"
-    alt="Team members - invite and select a role"
-    width="475px"
-  />
+{' '}
```
Comment (janpio) [inline content/800-data-platform/400-members/100-add.mdx:37]: ??

Comment (ruheni) [reply content/800-data-platform/400-members/100-add.mdx:37]: Not sure how got there or why it was added there😕

**Pull request merged**